README.md

```diff
@@ -353,7 +353,7 @@ This is how it works:
 |Function|Description|Documentation|
 |:------:|:----------|:-----------:|
 |**Collect**|Multiple independent data collection workers are collecting metrics from their sources using the optimal protocol for each application and push the metrics to the database. Each data collection worker has lockless write access to the metrics it collects.|[`collectors`](collectors/#data-collection-plugins)|
-|**Store**|Metrics are stored in RAM in a round robin database (ring buffer), using a custom made floating point number for minimal footprint.|[`database`](database/#database)|
+|**Store**|Metrics are stored between the RAM & the disk in an extremely efficient DBengine, using a custom made floating point number for minimal footprint.|[`database`](database/#database)|
```

Comment:
Hi @OdysLam ,

It is not clear for me the '&' in this description, is this a type error?

Reply:
I use it as a quick "and"

Reply:
I understand, when we are developing this is always used, but I do not know whether this is clear for everybody, I would prefer an "and" here.

Reply:
Agreed